registry.go

```diff
@@ -264,7 +264,7 @@ func (r *Registry) sendBatch(measurements []Measurement) {
 		} else {
 			sent := int64(0)
 			dropped := int64(0)
-			if resp.status == 200 {
+			if resp.status == 200 || resp.status == 202 {
```

Comment:
Makes sense but I'd update it so all 2xx are considered valid. Something like `if resp.status / 100 == 2`.

Note the test `TestRegistry_publish_errors` (file `registry_test.go` needs to be updated as well).

@copperlight is now the maintainer of the spectator thin clients, so adding him to the review.

Reply:
Note, a 200 and 202 should be the only 2xx responses that are used. A 202 indicates a partial failure and that some datapoints in the payload were invalid and rejected.